yarn.lock

```diff
@@ -2,524 +2,6114 @@
 # yarn lockfile v1
 
 
-"@kwsites/file-exists@^1.1.1":
```

Comment:
I don't see any changes to package.json, should this have changed?

Reply:
It turns out that it has the same content as the one in the workspace. I took a further look and found that if we removed the ```	"workspaces": [
		"source/wp-content/themes/wporg-gutenberg"
	]``` in ```source/wp-content/themes/wporg-gutenberg/package.json``` as well as its corresponding `yarn.lock` in the same folder and then `yarn` in `source/wp-content/themes/wporg-gutenberg`, the `yarn.lock` would only be generated in the root folder (the one we're looking at), and I'm thinking that this is the way workspace saves space from duplicating packages (by hoisting packages to workspace root). So perhaps we only need one yarn.lock here. Not so sure if I understand it correctly.

Reply:
Hmm the workspace setup confuses me a bit too, @StevenDufresne do we have a workspace setup for the theme in case one day we need a workspace for a plugin too? I've seen that in other repos

Reply:
Yeah, it needs some cleanup. The workspace declaration should be removed from the theme and the packages added from the root. We should probably open a new ticket.

Reply:
Ok. I'll reset the commit and open a new ticket for it. Other than that, do you think this PR is good to go?

Reply:
It doesn't appear like all the styles are loading properly, a few things I see:

- Buttons have different `border-radius`
- The footer has a huge space below it.

Reply:
> Buttons have different border-radius

Which one you're referring to? I saw all buttons on the sandbox having 2px `border-radius`, the same as production.

**Sandbox**
![Screen Shot 2022-09-07 at 3 42 49 AM](https://user-images.githubusercontent.com/18050944/188724983-044d69fe-f795-42b3-99f5-f9e5dcaecc94.png)
**Production**
<img width="346" alt="Screen Shot 2022-09-07 at 3 43 15 AM" src="https://user-images.githubusercontent.com/18050944/188725046-a6bcc740-a557-4db4-b3d5-1452d9ec4239.png">

> The footer has a huge space below it.
Fixed.

> It doesn't appear like all the styles are loading properly

Actually, the difference I spot is that the `font-weight` of `h1` and `h2` is `600` but not `400` (prod) 
and the `font-size` of some `p` is smaller than prod. 
I've found that the styles have already been imported, but is overridden by styles that I can't really tell where they are from. But it can be fixed with a tricky way, I've tested it and it worked, but before really pushing it, I'd like to know if there are other ideas.

Reply:
| `wp_is_mobile() = false` | `wp_is_mobile() = true`   |
| --- | --- |
|  <img width="425" alt="Screen Shot 2022-09-07 at 12 14 17 PM" src="https://user-images.githubusercontent.com/1657336/188780730-d2d5c4d8-20cf-4ccf-873e-e911d0dff50a.png"> |  <img width="459" alt="Screen Shot 2022-09-07 at 12 11 39 PM" src="https://user-images.githubusercontent.com/1657336/188780715-010b72c2-c09a-4e60-8d55-93828b04e594.png"> |
|   <img width="739" alt="Screen Shot 2022-09-07 at 12 14 07 PM" src="https://user-images.githubusercontent.com/1657336/188780744-eea73ca3-fa85-4fef-9174-6e61a3384cb7.png"> | <img width="730" alt="Screen Shot 2022-09-07 at 12 10 57 PM" src="https://user-images.githubusercontent.com/1657336/188780740-0531c950-bcf5-46d2-9081-1d77ffc386a4.png"> |

Reply:
I think the best we can do is to enqueue a mobile style sheet that fixes the differences. 

Additionally, the `<body>` tag needs a white background.

Reply:
I don't see the grey background or the space between the line and the button.
And for the `border-radius` of the button, I only saw it locally, not on the sandbox.
Here's the [fix](https://github.com/WordPress/wporg-gutenberg/pull/47/commits/1c6f5941e7a306e9dafe0da7f13a436bb26c7236#diff-589d2da6f7eebb3940a76caf617ade94e46f7ea2d62eff51da238d8b9f75f3cdR57) for it, now the local would also show the same `border-radius` as the sandbox and prod.
| Sandbox | Local | Prod |
| -----| ----------| -----|
|  ![Screen Shot 2022-09-08 at 3 39 26 AM](https://user-images.githubusercontent.com/18050944/188966712-2fba3b43-9f68-48ef-bb65-063ebe8f3f6c.png) | <img width="958" alt="Screen Shot 2022-09-08 at 3 39 52 AM" src="https://user-images.githubusercontent.com/18050944/188966726-70b01b65-4674-4ee5-9533-759068b1d381.png"> | ![Screen Shot 2022-09-08 at 3 41 06 AM](https://user-images.githubusercontent.com/18050944/188966734-febcbd11-f3e2-4aa6-8720-bcc3120d2a99.png) |